app/processing/get-payment-request-by-invoice-frn.js

```diff
@@ -3,7 +3,6 @@ const db = require('../data')
 const getPaymentSchemeByInvoiceAndFrn = async (invoiceNumber, frn) => {
   return db.completedPaymentRequest.findOne({
     where: { invoiceNumber, frn },
```

Comment:
why are these attributes no longer needed?

Reply:
Because in the events we want to send the full payment request.  Those attributes filter a lot of the properties out so needed to remove them.